arviz/data/inference_data.py

```diff
@@ -360,6 +366,9 @@ def from_netcdf(
         regex : bool, default False
             Specifies where regex search should be used to extend the keyword arguments.
             This feature is currently experimental.
+        base_group : str, default "/"
+            The group in the netCDF file where the InferenceData is stored. By default, assumes that the file
+            only contains an InferenceData object.
```

Comment:
```suggestion
            The group in the netCDF file where the InferenceData is stored. By default,
            assumes that the file only contains an InferenceData object.
```

this is the line being flagged as too long by pylint.